src/screens/FeedScreen/Header.js

```diff
@@ -484,26 +484,26 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: 14,
     lineHeight: 24,
-    color: colors.black
+    color: COLORS.black
   },
   textComment: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
     lineHeight: 18,
-    color: colors.gray
+    color: COLORS.gray8
   },
   usernameComment: {
     fontFamily: fonts.inter[500],
     fontWeight: '900',
     fontSize: 12,
     lineHeight: 24,
-    color: colors.black
+    color: COLORS.black
   },
   usernameTextComment: {
     fontFamily: fonts.inter[500],
     fontSize: 12,
     lineHeight: 24,
-    color: colors.gray
+    color: COLORS.gray8
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated from `colors.black` and `colors.gray` to `COLORS.black` and `COLORS.gray8` respectively. These changes are consistent with the update in the import statement.
```diff
- color: colors.black
+ color: COLORS.black
...
- color: colors.gray
+ color: COLORS.gray8
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->